libs/dart/seed_cryptography/lib/src/seed_cryptography_service.dart

```diff
@@ -0,0 +1,151 @@
+// ignore_for_file: constant_identifier_names
+
+import 'dart:convert';
+import 'dart:typed_data';
+
+import 'package:affinidi_tdk_cryptography/affinidi_tdk_cryptography.dart';
+import 'package:affinidi_tdk_seed_cryptography/src/exceptions/unsupported_seed_exception.dart';
+import 'package:affinidi_tdk_seed_cryptography/src/models/wallet_materials.dart';
+import 'package:convert/convert.dart';
+
+import 'exceptions/seed_decryption_exception.dart';
+
+/// A service class responsible for handling cryptographic operations
+/// related to wallet seed. This class provides methods to encrypt,
+/// decrypt, and manage cryptographic keys for secure data handling.
+///
+/// Example usage:
+/// ```dart
+/// final seed = Uint8List.fromList([1, 2, 3, 4, 5, 6, 7, 8]);
+/// final passphrase = 'mySecretPassphrase';
+///
+/// // Encrypt the seed
+/// final walletMaterials = await SeedCryptographyService.encryptSeed(
+///   seed: seed,
+///   passphrase: passphrase,
+/// );
+///
+/// // Decrypt the seed
+/// final decryptedSeed = await SeedCryptographyService.decryptSeed(
+///   encryptedSeedHex: walletMaterials.encryptedSeed,
+///   encryptionKeyHex: walletMaterials.encryptionKey,
+/// );
+/// ```
+class SeedCryptographyService {
```

Comment:
`Service` suffix is unnecessary as this only exposes some utility static functions.